docs/repo-docs/reference/ls.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: ls
+description: API reference for the `turbo ls` command
+---
+
+import { ExperimentalBadge } from '#/components/experimental-badge';
+
+List packages in your monorepo.
+
+```bash title="Terminal"
+turbo ls [package(s)] [flags]
+```
+
+When scoped to the entire repository, output includes package manager, package count, and all package names and directories.
+
+```bash title="Terminal"
+turbo ls
+```
+
+When scoped to one or more packages, output includes package name, directory, internal dependencies, and all tasks.
+
+```bash title="Terminal"
+turbo ls [package(s)]
```

Comment:
Up to you, but I think specifying a fake package name here is clearer than leaving the arg placeholder
```suggestion
turbo ls acme-pkg
```

Reply:
I agree, that was my first instinct as well but then noticed we didn't really do examples for other docs pages. I'll add it here and then we can always remove later if we want to.